packages/remix/src/index.ts

```diff
@@ -0,0 +1,22 @@
+import type { ActionFunctionArgs, LoaderFunctionArgs } from "@remix-run/node";
+import { createFrames as coreCreateFrames } from "frames.js/core";
+export { Button, type types } from "frames.js/core";
+
+export { fetchMetaData } from "./fetchMetaData";
+
+// @todo provide better type that can change returned handler type
+export const createFrames: typeof coreCreateFrames =
+  function createFramesForNextJS(options: any) {
```

Comment:
is this a typo - why are we naming this for next.js? is it because it's using WebAPI `Request`?

Reply:
this is just to make sure which function throws and error in error stack if something goes wrong.

Reply:
I will fix that after we merge since I will also have to change types for express so I can document what is the result of calling `createFrames`.